micrometer-core/build.gradle

```diff
@@ -81,6 +82,8 @@ dependencies {
     }
 
     // Aspects
+    implementation libs.aspectjrt
```

Comment:
I don't think we want to add a transitive dependency for micrometer-core users (same for micrometer-observation). When/how is this dependency used?

Reply:
Oh good catch, @mihalyr @kriegaex can you try to shed more light on this?